cmd/podman/rm.go

```diff
@@ -40,7 +41,7 @@ func init() {
 	rmCommand.SetUsageTemplate(UsageTemplate())
 	flags := rmCommand.Flags()
 	flags.BoolVarP(&rmCommand.All, "all", "a", false, "Remove all containers")
-	flags.BoolVarP(&rmCommand.Force, "force", "f", false, "Force removal of a running container.  The default is false")
+	flags.CountVarP(&rmCommand.Force, "force", "f", "Force removal of a running container. Used twice to force removal of a container with an invalid runtime configuration")
```

Comment:
Suggest: "Use twice (ff) to force the removal of a container with an invalid runtime configuration"